src/sage/features/mip_backends.py

```diff
@@ -84,7 +84,8 @@ def __init__(self):
         """
         JoinFeature.__init__(self, 'sage_numerical_backends_coin',
```

Comment:
I think it also makes sense to rename the feature now to `cbc` (if it's not used in too many places)

Reply:
right, I didn't rename the feature yet just to make the diff more readable. Will do